cmd/sebak/cmd/run.go

```diff
@@ -194,8 +197,11 @@ func parseFlagsNode() {
 		common.PrintFlagsError(nodeCmd, "--storage", err)
 	}
 
-	if threshold, err = strconv.Atoi(flagThreshold); err != nil {
-		common.PrintFlagsError(nodeCmd, "--threshold", err)
+	if signThreshold, err = strconv.Atoi(flagSIGNThreshold); err != nil {
```

Comment:
Same as previous question.

Reply:
@cmcm2222 As I understand our new proposer model, the voting will be done in 2 state transition, `INIT` -> `SIGN` and `SIGN` -> `ACCEPT`.

Reply:
But why would you want a different threshold ? That sounds quite dangerous.

Reply:
@Geod24 Hmm, it would be better node operator to set the threshold, but if it can be matter, how about limit the minimum threshold like 66%.

Reply:
@Geod24 So do you think the threshold option will be useless?

Reply:
At the moment, I don't see any reason for it to be configurable. In other words, [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

Reply:
@Geod24 If you mean only for SEBAK, the current threshold option is not important, it can be set in inside variable. :)

Reply:
My point is that we should just hardcode it to 67% for the time being and revise that when this kind of tweaks is needed (when we open the network).

Reply:
@Geod24 Yes, I agree. It can be hardcoded. If I have a right to decision, I will leave the threshold related options, it will be useful to control or test.